### DISCUSSION TITLE
No server is currently available to service your request or Sorry no resonse was returned

### DISCUSSION THREAD

**Discussion by Attila-Vigh** [community/community#112108] [Copilot Conversations]
### Select Topic Area

Show & Tell

### Body

Since this morning, the copilot started to write some answers then it stops or just returns an error as you can see in the next image. At this point the copilot is completely useless. Other times it takes ages to returns a response.
I tried to restart VSCode and my computer. Please help.

![image](https://github.com/community/community/assets/33238561/b6000dcf-e90d-4b56-a9a2-a8bce8c1c993)
![image](https://github.com/community/community/assets/33238561/5aa47935-8643-46e6-82c1-00f355a987e5)

**Comment by SimoneVL** [comment]
I changed the synchronisation to vsCode and it seems to work:

account --> synchronisation --> copilot github login

**Reply by HHUBRO** [reply]
it works！thanks！

**Reply by Attila-Vigh** [reply]
account? Mabey accounts. I have no idea where to find this `synchronisation --> copilot github login`
![image](https://github.com/community/community/assets/33238561/ad8cfed3-a088-432e-858e-26a83b3d6994)

**Reply by HHUBRO** [reply]
![image](https://github.com/community/community/assets/83570829/8f52af11-6d5b-4a56-9e20-317c5e2590e7)
![image](https://github.com/community/community/assets/83570829/fcf79db6-d667-49d2-91cc-c549394807ea)

**Reply by Attila-Vigh** [reply]
copilot github login ???
![image](https://github.com/community/community/assets/33238561/7186dc4d-6b43-4635-a93b-aaae82e0fa0e)

**Comment by HHUBRO** [comment]
I am facing a similar problem, it will output a small paragraph and then report an error

**Comment by BaptisteLecat** [comment]
After retrying several time it works. Im not facing the issue since 1 or 2 hours. Maybe a server issue in Github side
